src/gui/update.rs

```diff
@@ -1442,10 +1442,15 @@ pub fn handle_message(ajour: &mut Ajour, message: Message) -> Result<Command<Mes
             });
 
             ajour.catalog_categories_per_source_cache = categories_per_source;
+            let catalog_source_choice = if let Some(source) = &ajour.config.catalog_source {
```

Comment:
Not sure if this could be done prettier with a `.map()`.

Reply:
```rust
ajour
.config
.catalog_source
.map(CatalogSource::Choice)
.unwrap_or(CatalogSource::None)
```
?